portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getMessageFromServlet()">
```

Comment:
Just wanted to double check, we shouldn't be seeing any comments for now, right?

Reply:
Right now it's just printing out the contents of the static array. The next step will be to accept user input.